FirebaseAdmin/FirebaseAdmin/FirebaseException.cs

```diff
@@ -19,7 +19,7 @@ namespace FirebaseAdmin
     /// <summary>
     /// Common error type for all exceptions raised by Firebase APIs.
     /// </summary>
-    public sealed class FirebaseException: Exception
+    public class FirebaseException: Exception
```

Comment:
Lets keep this sealed for now and remove FirebaseAuthException. There's some work on our end to figure out the .NET error handling strategy.